blueprints/frontend-shared-config/files/.jscsrc

```diff
@@ -9,11 +9,14 @@
   "fileExtensions": [".js"],
   "esnext": true,
   "disallowEmptyBlocks": true,
+  "disallowMultipleSpaces": true,
```

Comment:
this one comes OOB [disallowmultiplespaces](http://jscs.info/rules.html#disallowmultiplespaces)